src/sql/azdata.proposed.d.ts

```diff
@@ -983,4 +983,23 @@ declare module 'azdata' {
 		 */
 		MsGraph = 7
 	}
+
+	export interface ResultSetSummary {
```

Comment:
It looks like we're not keeping this type in-sync with the same type in the dataprotocol client.  I guess that works since the client is just a "pass through" though ideally the same type would have the same definition in both places.  Should we update here as well? https://github.com/microsoft/sqlops-dataprotocolclient/blob/master/src/types.ts#L452

Reply:
you are right, some of the interfaces are not being referenced, instead of updating it, I think we should remove it, what do you think?